server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -446,6 +446,7 @@ private TableMgmtStats manageTablets(Iterator<TabletManagement> iter,
           state = newState;
         }
       }
+      tableMgmtStats.counts[state.ordinal()]++;
```

Comment:
Could this be moved to right after the computation of `state`?

Reply:
You are saying to move to right after line 421? Are you saying we don't need to capture the possible state change on line 446?